sdk/src/types/block/address/mod.rs

```diff
@@ -171,6 +171,24 @@ impl Address {
             (Self::ImplicitAccountCreation(address), _) => {
                 return Self::from(*address.ed25519_address()).unlock(unlock, context);
             }
+            (Self::Multi(address), Unlock::Multi(unlock)) => {
```

Comment:
Would you be fine with calling this local variable `multi_address`? Then it doesn't need to get shadowed by `address` withing the for-loop. Also `multi_address.len()` reads better than `address.len()`. Your choice though as this is just a preference.

Reply:
Done